### PR TITLE
[Enhancement] - Currency picker alert

### PR DESCRIPTION
**Pull request by Rcureton**
## What
- User can see the Currency Change Alert with the updated copy.
- Updated the strings from master.

## See 👀
oct-08-2018 16-14-11

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -95,6 +95,17 @@
     <item name="android:progressDrawable">@drawable/progress_bar_green_horizontal</item>
   </style>
 
+  <style name="AlertDialog" parent="Theme.AppCompat.Light.Dialog.Alert">
+    <item name="buttonBarPositiveButtonStyle">@style/AlertDialogButtonStyle</item>
+    <item name="buttonBarNegativeButtonStyle">@style/AlertDialogButtonStyle</item>
+    <item name="colorAccent">@color/text_secondary</item>
+  </style>
+
+  <style name="AlertDialogButtonStyle" parent="Widget.AppCompat.Button.ButtonBar.AlertDialog">
```
Comment (eoji) [inline app/src/main/res/values/styles.xml]: Sweet!

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -95,6 +95,17 @@
...
+    <item name="android:textColor">@color/ksr_green_500</item>
```
Comment (eoji) [inline app/src/main/res/values/styles.xml]: Should we refer to this is `colorAccent`?

File: app/src/main/res/layout/activity_account.xml [hunk 3]
```diff
@@ -84,11 +84,13 @@
...
           <Spinner
             android:id="@+id/currency_spinner"
+            style="@style/Base.Widget.AppCompat.Spinner.Underlined"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginEnd="@dimen/activity_horizontal_margin"
             android:layout_marginStart="@dimen/activity_horizontal_margin"
-            android:layout_marginTop="@dimen/grid_1">
+            android:layout_marginTop="@dimen/grid_1"
+            android:dropDownVerticalOffset="@dimen/grid_11">
```
Comment (eoji) [inline app/src/main/res/layout/activity_account.xml]: I don't think you'll need this because the drop down list is drawn over the spinner.

Comment (Rcureton) [reply app/src/main/res/layout/activity_account.xml]: Sounds good I'll remove it!

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (Rcureton) [reply app/src/main/res/values/styles.xml]: So it was ignoring the color when I used colorAccent so I went with TextColor

Comment (eoji) [reply app/src/main/res/values/styles.xml]: Sorry, I wasn't clear. I mean the color value you're passing in. Like this `<item name="android:textColor">@color/accent</item>`

**Comment by Rcureton** [comment]
[Currency picker alert](https://trello.com/c/rXIzObr0/979-currency-picker-alert)

**Pull request merged**